[issue] Fastembed build is failing

To reproduce: 
Create a sample cargo project:
$ cargo init
$ cargo add fastembed
$ cargo build

Error: 

```
   Compiling ort v2.0.0-rc.2
error[E0063]: missing fields `GetAliasMap`, `GetMayInplace`, `ReleaseAliasMap` and 1 other field in initializer of `OrtCustomOp`
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/ort-2.0.0-rc.2/src/operator/bound.rs:27:20
   |
27 |             implementation: ort_sys::OrtCustomOp {
   |                             ^^^^^^^^^^^^^^^^^^^^ missing `GetAliasMap`, `GetMayInplace`, `ReleaseAliasMap` and 1 other field

For more information about this error, try `rustc --explain E0063`.
error: could not compile `ort` (lib) due to 1 previous error
```

**Note:** 
Reason may be latest release from ort but fastembed do strict pinning.. not sure what is reason..

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,7 @@ homepage = "https://crates.io/crates/fastembed"
 anyhow = { version = "1" }
 hf-hub = {version="0.3", default-features = false, features = ["online"]}
 ndarray = { version = "0.15", default-features = false }
-ort = { version = "=2.0.0-rc.2", default-features = false, features = [ "ndarray" ] }
+ort = { version = "=2.0.0-rc.4", default-features = false, features = [ "ndarray" ] }
 rayon = { version = "1.10", default-features = false }
 serde_json = {version = "1"}
 tokenizers = { version = "0.19", default-features = false, features = ["onig"]}

